views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
@jwforres I'm not sure about this... `spec.host` is optional... if blank, the scheduler picks a node... there is still a `status.hostIP` which gets set when a pod actually gets scheduled onto a node. We probably want to keep showing status.hostIP under the `Status` section, and show `spec.host` (if set) outside the `Status` section